Update 20-scheduled-tasks.md

7underlines opened the pull request:
### 1. Why is this change necessary?
The override only works with config/packages/shopware.yml. 
This doesn't work: config/packages/shopware.yaml

### 2. What does this change do, exactly?
Fixes the documentation.

### 3. Describe each step to reproduce the issue or behaviour.


### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [-] I have written tests and verified that they fail without my change
- [-] I have squashed any insignificant commits
- [-] I have created a [changelog file](https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md) with all necessary information about my changes
- [+] I have written or adjusted the documentation according to my changes
- [-] This change has comments for package types, values, functions, and non-obvious lines of code
- [+] I have read the contribution requirements and fulfil them.

Phil23 commented:
Thank you @screeny05,

I've made a ticket for this. For now the tickets of the labs project are not public.

Thx! :thumbsup:

Phil23 commented:
Thank you  @screeny05,

was merged with 4e6ff07c639d4a4490dbd6cdac1e03cfa38c8eb2.

👍

7underlines committed:
Update 20-scheduled-tasks.md

The override only works with config/packages/shopware.yml. 
This doesn't work: config/packages/shopware.yaml

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/shopware/platform?pullRequest=1481) <br/>All committers have signed the CLA.

jleifeld commented:
We can´t reproduce that it doesn´t work with `.yaml`.
